doc/faq.md

```diff
@@ -139,6 +139,15 @@ When the whole page is rendered in your client's web browser it cannot respect y
 - **SEO:**
 The extenstion supports only backend search for regular search page and these results can be indexed by search engines like Google, Bing, etc... But because of the frontend implementantion instant search page results on category page cannot be indexed. But there is a workaround. Search parameters of the instant search page are pushed into page's URL. So it is possible to implement backend search base on the URL parameters so the instant search pages can be indexed. But the extension inself do not support this feature out of the box for now.
 
+## Can I have different ordering of products on category pages with instant search? (e.g. with Visual Merchandiser)
+
+Unfortunatelly this use-case is very tricky and not always achievable. Algolia is designed for searching and for providing the best relevancy on search queries. With this being said there are some limitations regarding sorting the results according specific attributes. 
+
+In case you have each product in only one category you can push to Algolia a custom attribute which reflects the product's ranking within it's category. Then you can just set this attribute as attribute for custom ranking and you are done.
+
+If you have product in multiple categories and within each category it has a different ranking it becomes tricky. This use-case can be handled only by slave indices when you basically have one index per category and in each of these indices you have a different ranking strategy.
```

Comment:
I would turn it a bit more positively.
Something like:
Doable but you need to be aware that you'll need to create more records.